apps/portals/adknowledgeportal/src/pages/ProjectDetailsPage.tsx

```diff
@@ -0,0 +1,176 @@
+import DetailsPage from '@sage-bionetworks/synapse-portal-framework/components/DetailsPage'
+import { DetailsPageContent } from '@sage-bionetworks/synapse-portal-framework/components/DetailsPage/DetailsPageContentLayout'
+import { DetailsPageContextConsumer } from '@sage-bionetworks/synapse-portal-framework/components/DetailsPage/DetailsPageContext'
+import ToggleSynapseObjects from '@sage-bionetworks/synapse-portal-framework/components/ToggleSynapseObjects'
+import { useGetPortalComponentSearchParams } from '@sage-bionetworks/synapse-portal-framework/utils/UseGetPortalComponentSearchParams'
+import { ColumnSingleValueFilterOperator } from '@sage-bionetworks/synapse-types'
+import React from 'react'
+import {
+  computationalSql,
+  experimentalModelsSql,
+  peopleSql,
+  projectsSql,
+  publicationsSql,
+  studiesSql,
+  targetEnablingResourcesDetailsPageSql,
+} from 'src/config/resources'
+import {
+  COMPUTATIONAL_TOOLS_COLUMN_NAMES,
+  computationalCardConfiguration,
+} from 'src/config/synapseConfigs/computational_tools'
+import {
+  EXPERIMENTAL_MODELS_COLUMN_NAMES,
+  experimentalDetailsTableConfigurationColumnLinks,
+  experimentalToolsCardConfiguration,
+} from 'src/config/synapseConfigs/experimental_tools'
+import { PEOPLE_COLUMN_NAMES } from 'src/config/synapseConfigs/people'
+import {
+  PROJECT_TABLE_COLUMN_NAMES,
+  projectCardConfiguration,
+} from 'src/config/synapseConfigs/projects'
+import {
+  publicationCardProps,
+  PUBLICATIONS_TABLE_COLUMN_NAMES,
+} from 'src/config/synapseConfigs/publications'
+import {
+  STUDY_TABLE_COLUMN_NAMES,
+  studyCardConfiguration,
+} from 'src/config/synapseConfigs/studies'
+import {
+  TARGET_ENABLING_RESOURCES_COLUMN_NAMES,
+  targetEnablingResourcesCardConfiguration,
+} from 'src/config/synapseConfigs/target_enabling_resources'
+import {
+  CardContainerLogic,
+  StandaloneQueryWrapper,
+  SynapseConstants,
+} from 'synapse-react-client'
+
+export default function ProjectDetailsPage() {
+  const searchParams = useGetPortalComponentSearchParams()
+
```

Comment:
Here I can envision adding logic to define properties to hoist into the `head` that would be meaningful for SEO. 

This would require some thought on each DetailsPage. For example, ADKP projects here can be queried by either the project name, or the grant number. Only one of those would define the 'canonical URL'.

Reply:
Could you add a TODO here in the code or a comment (I know some people don't like TODOs checked into the codebase)?

Reply:
We would probably add one for every details page, so that's a lot of TODOs. I think it's implicit in the task of trying to enrich the metadata on these pages, so I'm inclined to leave out the TODOs for now